### PR TITLE
Feature/new docker swarm

### PR DESCRIPTION
**Pull request by sashkachan**
NB:
- Can't use Weave Net because global plug-ins are not supported at this stage
- Can't use Weave Scope because running privileged containers is not supported either (https://github.com/docker/swarmkit/issues/1030#issuecomment-232299819)

### CODE REVIEW

File: deploy/swarmkit/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+# Overview
+
+This setup includes 3 nodes for Docker Swarm.
+master1 - is the Docker Swarm manager node
+node1 and node2 - worker nodes
+
+# Pre-requisities
+
+* Vagrant
```
Comment (pidster) [inline deploy/swarmkit/README.md:22]: What about Docker for Mac?

**Comment by pidster** [comment]
Please separate installation (set up environment) from deployment (deploy containers) instructions.  It should be possible for a Docker for Mac user to run the deployment sequence without needing Vagrant.

File: deploy/swarmkit/README.md (hunk 1 above)
Comment (sashkachan) [inline deploy/swarmkit/README.md:22]: Added docker-for-mac section. This will be limited to one node only, but much quicker to set up.

**Comment by philwinder** [comment]
Please rebase and force push this. 70+ commits going into the repo will look nasty in the log.

**Comment by pidster** [comment]
Or squash the commits?

**Comment by sashkachan** [comment]
Cleaned it up a bit!

**Pull request merged**